tests/remote/tests/API/2/FileTest.php

```diff
@@ -281,7 +284,9 @@ public function testAddFileFull() {
 		);
 	}
 	
-	
+	/**
+	 * @group api-sync
```

Comment:
Let's call this `classic-sync` — we call what 5.0 does "API syncing". (I know this won't include the classic-sync tests in `Sync/`, but that's OK, since the purpose of this is just to run against `API/` and exclude tests that use classic sync.)